libzpc.spec

```diff
@@ -47,7 +47,7 @@ The %{name}-static package contains the static library of %{name}.
 
 
 %prep
-%setup -q -n %{name}
+%autosetup -q -n %{name}-%{version}
```

Comment:
This should be just %autosetup, the default behaviour is `-q -n ...`.

Reply:
Done.